tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -750,11 +750,6 @@ EOF
 		tar xvJpf ${COCO_GUEST_COMPONENTS_TARBALL} -C ${ROOTFS_DIR}
 	fi
 
-	if [ "${MEASURED_ROOTFS}" == "yes" ]; then
```

Comment:
yeah, I didn't mention on the other PR but I wasn't so comfortable with this checking. Measure rootfs is a feature that we likely need to revisit. I'm glad that you changed it ;)